docs/architecture/seo.rst

```diff
@@ -17,11 +17,19 @@ Meta keywords are not used as they are ignored by all major search engines becau
 Meta description will be set to the product's description field. This does not affect the search engine ranking but it affects the snippet of text shown along with the search result.
 
 
-JSON-LD
+Structured Data
 -------
 
-All product pages contain semantic descriptions in JSON-LD `Structured Data <https://developers.google.com/search/docs/guides/intro-structured-data>`_ format.
+Homepage and product pages contain semantic descriptions in JSON-LD `Structured Data <https://developers.google.com/search/docs/guides/intro-structured-data>`_ format.
 
 It does not directly affect the search engine ranking but it allows search engines to better understand the data ("this is a product, it's available, it costs $10").
 
-It allows search engines like Google to show product photos, prices, ratings etc. along with their search results.
+It allows search engines like Google to show product photos, prices, availability, ratings etc. along with their search results.
+
+
+Open Graph
```

Comment:
Is this really SEO? It seems like a new file would be a better fit. Maybe under integrations rather than architecture?

Reply:
It won't affect on-page SEO directly, but will improve links performance on Social Media, which may be one of the factors to rank page higher. Technically, Page Graph is part of SMO (Social Media Optimization) strategy. 

However, I think that folks would rather look for it under `SEO` than integrations. 
Maybe we can work it out within this section (file?) eg. renaming section to better fit its content, or creating dedicated chapter for SMO

Reply:
We've decided to move SEO and SMO sections to integrations